ipi_tests/regression_tests/runstools.py

```diff
@@ -65,17 +65,25 @@ def run(self, cwd, nid):
             if nl > 1:
                 self.files.append(line.split()[0])
                 self.forms.append(line.split()[1])
+
                 if len(line.split()) > 2:
-                    listll = []
+
+                    listcol = []
                     listtol = []
-                    for ll in line.split()[2:]:
-                        lltol = ll.split(",")
-                        listll.append(int(lltol[0]))
-                        if len(lltol) > 1:
-                            listtol.append((float(lltol[1]), float(lltol[2])))
-                        else:
-                            listtol.append(None)
-                    self.usecol.append(listll)
+                    info_tol = line.split()[2].split(",")
+                    if info_tol[0] == "all":
+                        listcol = info_tol[0]
+                        listtol.append((float(info_tol[1]), float(info_tol[2])))
```

Comment:
I think you should write it such that it also supports the case in which you have "all" without tolerances. It is a bit redundant, but it is more consistent with the structure when you can specify columns also with default tolerances